### DISCUSSION TITLE
Combine auto-rotate with show dimensions (as SVG lines)

### DISCUSSION THREAD

**Discussion by virtualheritage** [google/model-viewer#4445] [Q&A]
Hi, I would like to show a model with dimensions as svg lines (like the example shown in documentation: [https://modelviewer.dev/examples/annotations/index.html#dimensions](https://modelviewer.dev/examples/annotations/index.html#dimensions)) and auto-rotate it, but if I configure the model to auto-rotate, svg lines won't follow the rotation as annotations do.

I guess it's beacuse 'camera-change' listener doesn't get trigger when model auto-rotates, so svg lines aren't being updated. Is there any other event listener or alternative I could use?

**Comment by elalish** [comment]
Good question! I think all we can do is poll the turntable rotation: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-properties-turntableRotation

Maybe we should have an event for that too?
